amundsen_application/static/js/components/ColumnList/ColumnStats/index.tsx

```diff
@@ -24,12 +24,12 @@ const ColumnStatRow: React.FC<ColumnStatRowProps> = ({
   stat_type,
   stat_val,
 }: ColumnStatRowProps) => {
-  const numberVal = +stat_val;
-
   return (
     <div className="column-stat-row">
       <div className="stat-name body-3">{stat_type.toUpperCase()}</div>
-      <div className="stat-value">{formatNumber(numberVal)}</div>
+      <div className="stat-value">
+        {isNumber(stat_val) ? formatNumber(+stat_val) : stat_val}
+      </div>
```

Comment:
Could we extend the tests of ColumnStats to cover for this logic?

Reply:
I tested the two functional individually wasn't sure if I should add tests to ColumsStats because I though those might be too specific to the logic rather than the presentational aspect of the component.

Reply:
I see, however, if we ever change our logic to figure this out, this part would not be tested.

Reply:
I see what you mean, so I should check if the correct text is displayed then?

Reply:
Yes, please!